src/EnergyPlus/WindowManager.cc

```diff
@@ -7324,6 +7438,7 @@ namespace WindowManager {
         int Layer;
         int BlNum; // Blind number
         int i;
+        // initialize to some non-realistic values, which enables checking of whether it's first executed
```

Comment:
Does this comment still apply?

Reply:
No, this is some old comment. I just removed it.